kuma/users/migrations/0012_auto_20190914_2209.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-09-14 22:09
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+DISCARDED_LOCALES = (
+    'af', 'az', 'cs', 'ee', 'ff', 'fy-NL', 'ga-IE', 'ha', 'hr', 'ig', 'ka',
+    'ln', 'mg', 'ml', 'ro', 'son', 'sq', 'sr', 'sr-Latn', 'sw', 'ta', 'te',
+    'tl', 'tn', 'wo', 'xh', 'yo', 'zu'
+)
+
+
+def update_users_with_discarded_locales(apps, schema_editor):
+    """
+    Nullify the preferred locale of all users with one of the DISCARDED_LOCALES.
+    """
+    User = apps.get_model('users', 'User')
+    User.objects.filter(locale__in=DISCARDED_LOCALES).update(locale=None)
```

Comment:
Instead of listing these, as this'll forever keep these strings in git, you could do:
```python
from django.conf import settings
User.objects.exclude(locale__in=settings.ACCEPTED_LOCALES).update(locale=None)
```

Reply:
I'd prefer to keep it as is, since it captures exactly what this data migration should do at this point in time. The value of `settings.ACCEPTED_LOCALES` will change over time, changing what this migration does (although it's probably safe as long as we don't add any of these locales back?).

Reply:
I'm totally cool with that. Mind you, `settings.ACCEPTED_LOCALES` does change but the migration will never ever run again in stage or prod because they're remembered as run (past tense).